docs/source/quicktour.mdx

```diff
@@ -81,7 +88,7 @@ GPUs.
 
 ### Model Arguments
 
-The `--model_args` argument takes a string representing a list of model
+The `model-args` argument takes a string representing a list of model
```

Comment:
But model-args is no longer an explicit key, right?

Reply:
yes ! that's why I removed the `--`, however, it's still an argument named `model-args` so this explaination still stands